lib/core/src/Cardano/Wallet/Api/Types.hs

```diff
@@ -621,10 +664,26 @@ walletDelegationOptions = taggedSumTypeOptions $ TaggedObjectOptions
 -- >>> Aeson.encode $ Restoring (Quantity 14)
 -- {"status":"restoring","progress":{"quantity":14,"unit":"percent"}}
 syncProgressOptions :: Aeson.Options
-syncProgressOptions = taggedSumTypeOptions $ TaggedObjectOptions
-    { _tagFieldName = "status"
-    , _contentsFieldName = "progress"
-    }
+syncProgressOptions = taggedSumTypeOptions defaultSumTypeOptions $
+    TaggedObjectOptions
+        { _tagFieldName = "status"
+        , _contentsFieldName = "progress"
+        }
+
+-- | Options for encoding a synchronization progress. It can be serialized to
```

Comment:
> -- | Options for encoding a synchronization progress. 

Should be one of:

* `-- | Options for encoding synchronization progress. `
* `-- | Options for encoding a synchronization progress object.` <-- Preferred.